### PR TITLE
Convert from drawer layout to tab layout

### PR DESCRIPTION
**Pull request by tiembo**
Note: the toolbars will be re-added (with new designs) in future CLs.

Screenshot_1560977670
Screenshot_1560977637
Screenshot_1560977640

### CODE REVIEW

File: app/src/main/java/com/google/samples/apps/sunflower/adapters/SunflowerPagerAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const val MY_GARDEN_PAGE_INDEX = 0
+const val PLANT_LIST_PAGE_INDEX = 1
+
+class SunflowerPagerAdapter(
+    private val context: Context,
+    fragmentManager: FragmentManager
+) : FragmentStatePagerAdapter(fragmentManager, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
+
+    override fun getCount() = 2
```
Comment (BatmanInGotham2) [inline app/src/main/java/com/google/samples/apps/sunflower/adapters/SunflowerPagerAdapter.kt]: Maybe a constant/more descriptive function name to describe what the '2' is

File: app/src/main/res/navigation/nav_garden.xml [hunk 2]
```diff
@@ -18,43 +18,21 @@
...
-    app:startDestination="@+id/garden_fragment">
+    app:startDestination="@+id/view_pager_fragment">
 
     <fragment
-        android:id="@+id/garden_fragment"
-        android:name="com.google.samples.apps.sunflower.GardenFragment"
-        android:label="@string/my_garden_title"
-        tools:layout="@layout/fragment_garden">
+        android:id="@+id/view_pager_fragment"
+        android:name="com.google.samples.apps.sunflower.ViewPagerFragment"
```
Comment (BatmanInGotham2) [inline app/src/main/res/navigation/nav_garden.xml]: should this live in the root folder or a subfolder within com.google.samples.apps.sunflower?

Comment (tiembo) [reply app/src/main/res/navigation/nav_garden.xml]: It's in the same folder as the other Fragments.

File: app/src/main/java/com/google/samples/apps/sunflower/adapters/SunflowerPagerAdapter.kt (hunk 1 above)
Comment (tiembo) [reply app/src/main/java/com/google/samples/apps/sunflower/adapters/SunflowerPagerAdapter.kt]: Updated

**Pull request merged**